00 - Contribute to the Obsidian Hub/03 Contributor Notes/03.02 Design Decisions/hub.yaml proposal for themes.md

```diff
@@ -0,0 +1,303 @@
+        publish: true
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        gutter:
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        print:
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        callouts:
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        checklists:
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        images: []
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        tables: []
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        asides: []
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        highlights: []
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        color_schemes: []
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        snippets: [] # Path to css snippets relative to the root of the repo. We could parse the css similar to style settings
```

Comment:
I think it's worth considering two types of snippets:
1. snippets to enhance/change the theme (requires the user using the theme)
2. snippets to use a certain theme feature in a different theme

Reply:
Maybe this we can tackle later? Or as a separate categorization of snippets